upload/views.py

```diff
@@ -185,19 +185,20 @@ def assign(request):
         )
     elif package_id:
         package = get_object_or_404(Package, pk=package_id)
-
-        if package.assignee is None:
-            package.assignee = user
-            package.save()
-
+        is_reassign = package.assignee is None
+        
+        package.assignee = user
+        package.save()
+        
+        if is_reassign:
```

Comment:
@rafaelpezzuto não entendi

Reply:
@robertatakenaka significa "é reatribuição"? outro termo que ajudaria a clarificar seria `has_assignee` (tem usuário atribuído?).

Caso tenha usuário atribuído, um warning é informado na tela do usuário indicando que o pacote foi reatribuído (saiu de um usuário e foi para outro).

Reply:
entendi... a lógica está correta, mas o nome da variável dá confusão, pois `is_reassign` fica como `True` se antes da atribuição o pacote não está atribuído a ninguém.
Ou o nome da variável deveria ser `is_assign` ou a linha 188 deveria ser `is_reassign = package.assignee is not None` (indicando que é uma ação de reatribuição), não é?

Reply:
@robertatakenaka adicionei o `not` que esqueci para os nomes fazerem sentido.